modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/resources.jsp

```diff
@@ -35,6 +35,34 @@ String inlineEditSaveURL = GetterUtil.getString((String)request.getAttribute(CKE
 	long javaScriptLastModified = PortalWebResourcesUtil.getLastModified(PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR);
 	%>
 
+	<script data-senna-track="temporary" type="text/javascript">
+		window.CKEDITOR_GETURL = function (resource) {
+			var languageParameter = '&languageId=' + themeDisplay.getLanguageId();
```

Comment:
Maybe we should use `URL` and `searchParams`/`append` to avoid the manual string manipulation in this function.

Reply:
Yeah I thought about it, but just copied CKEditor's `getUrl` function, I'll look into it